spec/donation_system/adapters/resulting_adapted_payment_api.rb

```diff
@@ -64,11 +64,11 @@ module Adapters
       end
 
       it 'responds to last four digits of the card' do
-        expect(payment_data.last4).to eq('4242')
+        expect(payment_data.respond_to?(:last4)).to be_truthy
       end
 
       it 'responds to brand' do
-        expect(payment_data.brand).to eq('Visa')
+        expect(payment_data.respond_to?(:brand)).to be_truthy
```

Comment:
Why not `be true` instead of `be_truthy` if `respond_to?` only returns a boolean answer? Same for the test above this one `responds to last four digits of the card`.

Reply:
Good point!